images/base/Dockerfile

```diff
@@ -7,6 +7,9 @@
 FROM centos:centos7
 
 # components from EPEL must be installed in a separate yum install step
+# workaround systemd upgrade path problems yank this out as soon as centos7.2
```

Comment:
make this a `# TODO: systemd update from centos 7.1 -> 7.2 is broken, remove this once 7.2 base images land`